src/scenes/NftDetailPage/NftDetailNote.tsx

```diff
@@ -0,0 +1,241 @@
+
+  const [collectorsNote, setCollectorsNote] = useState(nftCollectorsNote ?? '');
+  const unescapedCollectorsNote = useMemo(() => unescape(collectorsNote), [collectorsNote]);
+
+  const hasCollectorsNote = useMemo(() => collectorsNote.length > 0, [collectorsNote]);
+
+  const collectorsNoteRef = useRef<HTMLDivElement>(null);
+
+  const scrollDown = () => {
+    if (collectorsNoteRef.current) {
+      collectorsNoteRef.current.scrollIntoView({
+        block: 'end',
+        inline: 'nearest',
+        behavior: 'smooth',
+      });
+    }
+  };
+
+  const handleEditCollectorsNote = useCallback(() => {
+    setIsEditing(true);
+
+    // TODO Expand note to full height first time it is opened
+    // Currently user has to type first to expand it fully
+
+    // Scroll down - wait 100ms so that element exists before scrolling to bottom of it
+    setTimeout(() => {
+      scrollDown();
+    }, 200);
+  }, []);
+
+  const updateNft = useUpdateNft();
+
+  const handleSubmitCollectorsNote = useCallback(async () => {
+    setGeneralError('');
+
+    if (unescapedCollectorsNote.length > MAX_CHAR_COUNT) {
+      // No need to handle error here, since the form will mark the text as red
+      return;
+    }
+
+    setIsEditing(false);
+
+    try {
+      await updateNft(nftId, collectorsNote);
+    } catch (error: unknown) {
+      if (error instanceof Error) {
+        setGeneralError(formatError(error));
+      }
+    }
+  }, [updateNft, nftId, collectorsNote, unescapedCollectorsNote]);
+
+  // If the user hits cmd + ctrl enter, submit the note
+  const handleKeyDown = useCallback(
+    async (event) => {
+      if (event.key === 'Enter' && event.metaKey) {
+        event.preventDefault();
+        await handleSubmitCollectorsNote();
+      }
+    },
+    [handleSubmitCollectorsNote]
+  );
+
+  const handleNoteChange = useCallback((event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setCollectorsNote(event.target?.value);
+    setNoteHeight(event.target?.scrollHeight);
+
+    // On clear, reset note height (textarea scrollHeight does not decrease on its own, it only increases)
+    // So we use this hard reset if the user deletes all content. Could have more elegant solution
+    // TODO Reduce size to text content on any delete
+    if (event.target?.value === '') {
+      setNoteHeight(MIN_NOTE_HEIGHT);
+    }
+
+    // Scroll down as the user input goes off the screen
+    // Need setTimeout so that textarea height is updated
+    setTimeout(() => {
+      scrollDown();
+    }, 50);
+  }, []);
+
+  return (
+    <StyledContainer ref={collectorsNoteRef} tabIndex={0} onKeyDown={handleKeyDown}>
+      <Spacer height={18} />
+
+      <StyledTitleAndButtonContainer>
+        {(hasCollectorsNote || isEditing) && <BodyRegular>Collector&rsquo;s Note</BodyRegular>}
+        {userOwnsAsset && !hasCollectorsNote && !isEditing && (
+          <TextButton text={`+ Add Collector's Note`} onClick={handleEditCollectorsNote} />
```

Comment:
supernit: any reason to use back ticks here? can just do `text="+ Add Collector's Note"`